.github/DEVELOPMENT.md

```diff
@@ -270,6 +270,20 @@ with changes in each section arranged to have new features first, performance
 improvements second, and bugfixes third. See [the release notes for 395](https://github.com/trinodb/trino/pull/13975)
 as an example.
 
+Release notes which may include a breaking change to Trino are indicated with a
```

Comment:
I think it would be better to somehow add it to the actual docs so we can link to it easily. Still have to figure out how to do that as well btw .. maybe just a link around the emoji but then how do we get an alt/description text on the image .. maybe we need a replacement dev in conf.py to make it easier .. ping me to chat about that if needed

Reply:
Are you removing this here @colebow